src-ts/lib/breadcrumb/breadcrumb-item/BreadcrumbItem.tsx

```diff
@@ -11,7 +11,7 @@ interface BreadcrumbItemProps {
 
 const BreadcrumbItem: FC<BreadcrumbItemProps> = (props: BreadcrumbItemProps) => {
     return (
-        <li key={props.index}>
+        <li key={props.index} onClick={() => props.item.onClick && props.item.onClick(props.item)}>
```

Comment:
let's use optional chaining when possible
```
props.item.onClick?.(props.item)

Reply:
Thanks @brooketopcoder. I've updated the PR with the changes.